### PR TITLE
Add Card for STEM Diverse Tv Cms to projects list

### LINKED ISSUE #228: Bug: Add STEM diverse tv cms to projects list
### Describe the bug
Although [stem-diverse-tv-cms](https://github.com/anitab-org/stem-diverse-tv-cms) is actively developed, still it is not listed in the projects list

### To Reproduce
1. Go to https://anitab-org.github.io/
2. Click on Projects
3. Scroll down

### Expected behavior
[stem-diverse-tv-cms](https://github.com/anitab-org/stem-diverse-tv-cms) should also be listed in the projects list.

### Desktop (please complete the following information):
 - OS: Any
 - Browser: Any
 - Version: Any

### Smartphone (please complete the following information):
 - Device: Any
 - OS: Any
 - Browser: Any
 - Version: Any

### PR DESCRIPTION
**Pull request by ishitakeshawani**
### Description

Add Card for STEM diverse tv cms to the projects list.

Fixes #228 

### Type of Change:

- Code
- User Interface

### How Has This Been Tested?
https://ishitakeshawani.github.io/anitab-org.github.io/
anitab-org-gif

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have attached the link to the deployed site.

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: src/content/projects_content.js [hunk 1]
```diff
@@ -344,8 +344,36 @@ const projects = {
...
         },
+        {
+          image: {
+            source: require('./../assets/placeholder.png'),
+          },
+          title: 'Stem Diverse Tv Cms',
+          description:
+            'Content Management System (CMS) for managing content of STEM Diverse TV backend.',
+          labels: [
+            { title: 'flask', color: '#F8972B' },
```
Comment (Rahulm2310) [inline src/content/projects_content.js:355]: Remove `flask` and `python` and add `react js` and `javascript` labels.

**Comment by Rahulm2310** [comment]
@ishitakeshawani I see a lot of irrelevant file changes. Please remove these unnecessary changes from the commit.

File: src/content/projects_content.js [hunk 2]
```diff
@@ -344,8 +344,36 @@ const projects = {
           contributors_url:
             'https://api.github.com/repos/anitab-org/stem-diverse-tv/contributors',
         },
+        {
+          image: {
+            source: require('./../assets/placeholder.png'),
+          },
+          title: 'Stem Diverse Tv Cms',
```
Comment (Rahulm2310) [inline src/content/projects_content.js:351]: Stem Diverse TV CMS

File: src/content/projects_content.js [hunk 3]
```diff
@@ -344,8 +344,36 @@ const projects = {
...
+            {
+              title: 'zulip',
+              source: require('./../assets/zulip.png'),
+              url:
+                'https://anitab-org.zulipchat.com/#narrow/stream/225705-STEM-diverse-tv',
+            },
+            {
+              title: 'github',
+              source: require('./../assets/github.png'),
+              url: 'https://github.com/anitab-org/stem-diverse-tv',
```
Comment (Rahulm2310) [inline src/content/projects_content.js:368]: Update the url to https://github.com/anitab-org/stem-diverse-tv-cms

Comment (ishitakeshawani) [reply src/content/projects_content.js:368]: ok, I will do all the above changes that you suggested.

File: asset-manifest.json [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    "static/media/testing.png": "./static/media/testing.1d609eba.png",
+    "static/media/twitter_logo.png": "./static/media/twitter_logo.b696dd8e.png"
+  },
+  "entrypoints": [
+    "static/js/runtime-main.77c289f4.js",
+    "static/js/2.48d6caae.chunk.js",
+    "static/css/main.69eda509.chunk.css",
+    "static/js/main.3b76bc82.chunk.js"
+  ]
+}
```
Comment (keshakaneria) [inline asset-manifest.json:44]: Each file needs to have an empty line at the end of file as per the conventions.

**Review by keshakaneria — CHANGES_REQUESTED** [review]

**Comment by ishitakeshawani** [comment]
@Rahulm2310  @keshakaneria I have made all changes.

**Comment by Rahulm2310** [comment]
@ishitakeshawani Can't see your changes.

**Comment by keshakaneria** [comment]
> @Rahulm2310 @keshakaneria I have made all changes.

You might have not pushed the changes in the branch from which you sent this PR.

**Comment by ishitakeshawani** [comment]
yes, I opened PR from a gh-pages branch in which I was testing. I am closing this PR and will make another. Thank you @keshakaneria for To draw attention to it.

**Pull request closed without merging**